Library/Formula/opentsdb.rb

```diff
@@ -0,0 +1,104 @@
+require "socket"
+class Opentsdb < Formula
```

Comment:
Please leave an empty line between the `require` and first `class` here, Thanks.